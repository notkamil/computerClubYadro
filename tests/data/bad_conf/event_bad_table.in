3
08:00 20:00
10
09:00 1 client1
10:00 2 client1 10