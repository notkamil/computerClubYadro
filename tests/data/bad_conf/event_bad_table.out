10:00 2 client1 10